mono/mini/method-to-ir.c

```diff
@@ -7819,7 +7819,10 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 			ins_flag = 0;
 			constrained_class = NULL;
 			if (need_seq_point)
-				emit_seq_point (cfg, method, next_ip, FALSE, TRUE);
+			{
+				gboolean sym_seq_point = sym_seq_points && mono_bitset_test_fast (seq_point_locs, ip - header->code);
+				emit_seq_point (cfg, method, next_ip, FALSE,((sp == stack_start) || !sym_seq_point));
+			}
```

Comment:
Shouldn't this be sp != stack_start ?

Reply:
Hi, Vargaz, maybe you are right, now I checked if they are nested calls in a different way. Thanks for the review.